cvat-ui/src/components/annotation-page/attribute-annotation-workspace/attribute-annotation-sidebar/attribute-annotation-sidebar.tsx

```diff
@@ -50,6 +52,65 @@ interface LabelAttrMap {
     [index: number]: any;
 }
 
+const componentShortcuts = {
```

Comment:
Now in each component we have:

1. const componentShortcuts = { SHORTCUT_1: { ... }, SHORTCUT_2: { ... } }
2. const subKeyMap = { SHORTCUT_1: { ... }, SHORTCUT_2: { ... } } (actually the same as componentShortcuts)
3. const handlers = { SHORTCUT_1: () => {}, SHORTCUT_2: () => {} }

I will propose to generate subKeyMap dinamically, e.g.:
```
                <GlobalHotKeys
                    keyMap={Object.fromEntries(Object.keys(componentShortcuts).map((key) => [key, keyMap[key]]))}
                    handlers={handlers}
                />
```

And add types to `handlers`, e.g.:
```
const handlers: Record<keyof typeof componentShortcuts, (event?: KeyboardEvent) => void> = {
```

It will make as sure that we do not forget to declare any handlers. 

What do you think?

Reply:
Also, in this case we may prevent undefined values passed to <GlobalHotKeys> as discussed before:

```
Object.fromEntries(Object.keys(componentShortcuts).filter((key) => !!keyMap[key]).map((key) => [key, keyMap[key]]))
```

This code may be separated as an utility function.